### DISCUSSION TITLE
XFox111 Tabs Aside Extension Link

### DISCUSSION THREAD

**Discussion by sahil48** [tim-we/tabs-aside#116] [General]
@xfox111

Following up on https://github.com/tim-we/tabs-aside/issues/115, the other Tabs Aside Extension link it at:

https://github.com/xfox111/TabsAsideExtension

I was asking if maybe any collaboration between the two extensions was worthwhile.

**Comment by XFox111** [comment]
@sahil48

Hi, yeah, that's cool that there're more extensions with such functionality. Maybe we can collaborate in some way with @tim-we. However, I've just scrolled through his repo. Our extensions have quite different architectures, so I'm not sure if we can share our codebases with each other.

Anyway, I've kinda left my project and don't actively work on it, so if Tim wants, we can talk about maintainer role in my project. I'll be glad to work with him

**Reply by tim-we** [reply]
Hi @XFox111, nice to hear from another developer of a similar project. At the moment I am more focused on other projects but I plan to work on Tabs Aside again (I want to rewrite some parts of the architecture and use React for the UI). So yeah right now my project is in a similar state and I don't have time to contribute to your project at the moment, but perhaps in the future :)
